### PR TITLE
Persistent workers

### PR DESCRIPTION
**Pull request by fkorotkov**
Screen Shot 2020-12-16 at 11 57 06 AM
Screen Shot 2020-12-16 at 11 57 36 AM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,11 +13,15 @@
   "private": true,
   "license": "MIT",
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.32",
+    "@fortawesome/free-brands-svg-icons": "^5.15.1",
+    "@fortawesome/free-solid-svg-icons": "^5.15.1",
```
Comment (RDIL) [inline package.json:18]: This seems like something that will drastically pump up the bundle size, is it really necessary?  Can it just be done as static individual SVGs?  Or better yet, stick with material icons?

**Review by RDIL — CHANGES_REQUESTED** [review]
1. Hooks might be better for this, and 2:

File: package.json (hunk 1 above)
Comment (fkorotkov) [reply package.json:18]: Treeshaking mitigates it. Not everything is bundled.

Comment (RDIL) [reply package.json:18]: still, this seems like it would be more worth it to add the svg files themselves, and let svgr take care of them

Comment (fkorotkov) [reply package.json:18]: Sounds like a premature optimization. The package also provides an `ImageDefinition` type with the SVG path and width/height.

**Comment by fkorotkov** [comment]
Rewriting the code base to hooks is a separate problem. Still TBD how React, Relay and Material UI will work together.

**Pull request merged**